tests/dm_control/test_dm_control_policies.py

```diff
@@ -0,0 +1,51 @@
+import gc
+import unittest
+
+from dm_control.suite import ALL_TASKS
+import tensorflow as tf
+
+from garage.baselines import LinearFeatureBaseline
+from garage.envs.dm_control import DmControlEnv
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import GaussianMLPPolicy
+from garage.tf.spaces import Box
+
+
+class TestDmControlPolicies(unittest.TestCase):
```

Comment:
This test is unnecessary. We don't need to run an integration test with every dm_control environment. We should perhaps run one of these.

Reply:
So I should just keep the implementation of `test_dm_control_policy`? They are functionally equivalent.

Reply:
I think it's sufficient to test one dm_control environment against one algorithm from each backend.